pkgs/development/python-modules/pycritty/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "pycritty";
+  version = "0.3.5";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1lrmd4a1ps3h9z0pndfjfrd2qa7v3abd6np75fd2q2ffsqv7ar6x";
+  };
+
+  # The package does not include any tests to run
+  doCheck = false;
```

Comment:
Could we do a `pythonImportsCheck = [ "pycritty" ];`?

Reply:
@fabaff Added!